internal/test/e2e/migrations_test.go

```diff
@@ -137,19 +138,11 @@ func TestMigrations(t *testing.T) {
 		ress, err := b.Alerts(context.Background(), alerts.AlertsOpts{})
 		tt.OK(err)
 		for _, alert := range ress.Alerts {
-			// skip if not a migration alert
-			data, ok := alert.Data["objectIDs"].(map[string]interface{})
-			if !ok {
-				continue
-			}
-
-			// collect all object ids per bucket
-			for bucket, ids := range data {
-				if objectIDs, ok := ids.([]interface{}); ok {
-					for _, id := range objectIDs {
-						got[bucket] = append(got[bucket], id.(string))
-					}
-				}
+			var objects []api.ObjectMetadata
+			b, _ := json.Marshal(alert.Data["objects"])
+			json.Unmarshal(b, &objects)
```

Comment:
leftovers

Reply:
That's not a leftover. That's the easiest way to assert alert data without awkwardly casting it to something.